src/App.js

```diff
@@ -28,18 +37,23 @@ function App() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          DevCycle SDK Docs
+          DevCycle React SDK Docs
         </a>
       </div>
     </div>
   );
 }
 
+/**
+ * Wrapping your app with the DevCycle provider will make the client
+ * available to all child components.
+ */
 export default withDevCycleProvider({
   sdkKey: process.env.REACT_APP_DEVCYCLE_CLIENT_SDK_KEY,
   user: {
-    user_id: 'user1',
-    email: 'initial_user@email.com'
+    user_id: 'user-1',
+    name: 'User 1',
+    email: 'user1@email.com'
```

Comment:
is it possible to import this from User.js? might be confusing if the user changes 'User 1' here but not in that file

Reply:
good point — I'll move those to a shared file